spec/controllers/idv/cancellations_controller_spec.rb

```diff
@@ -118,6 +127,20 @@
       put :update, params: { step: 'first', cancel: 'true' }
     end
 
+    it 'logs cancellation go back with extra analytics attributes for barcode step' do
+      expect(@analytics).to receive(:track_event).with(
+        'IdV: cancellation go back',
+        step: 'barcode',
+        proofing_components: nil,
+        cancelled_enrollment: false,
+        enrollment_code: '123',
+        enrollment_id: 1,
```

Comment:
We shouldn't ever set `id` on a test object; just let it get assigned an id and then test for `enrollment.id`